### PR TITLE
Add web services used in Live Papers

### PR DESCRIPTION
**Pull request by apdavison**

### CODE REVIEW

File: instances/service/AllenInstituteCellTypesDataPortal.jsonld [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/AllenInstituteCelTypesDataPortal",
```
Comment (UlrikeS91) [inline instances/service/AllenInstituteCellTypesDataPortal.jsonld]: you miss a L in "cell" in "AllenInstituteCelTypesDataPortal"

File: instances/service/CollaboratoryLab.jsonld [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/CollaboratoryLab",
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Service",
+  "definition": "EBRAINS JupyterLab service",
```
Comment (UlrikeS91) [inline instances/service/CollaboratoryLab.jsonld]: Maybe rather:
"The 'Collaboratory Lab' is a web-based JupyterLab service provided by the EBRAINS research infrastructure."

File: instances/service/CollaboratoryLab.jsonld [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/CollaboratoryLab",
```
Comment (UlrikeS91) [inline instances/service/CollaboratoryLab.jsonld]: Should reflect the name of the instance: "/EBRAINSCollaboratoryLab"

File: instances/service/CollaboratoryWiki.jsonld [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/CollaboratoryWiki",
```
Comment (UlrikeS91) [inline instances/service/CollaboratoryWiki.jsonld]: Should reflect the name of the instance: "/EBRAINSCollaboratoryWiki"

File: instances/service/CollaboratoryWiki.jsonld [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Service",
+  "definition": "EBRAINS Collaboratory Wiki",
```
Comment (UlrikeS91) [inline instances/service/CollaboratoryWiki.jsonld]: Maybe rather:
"The 'Collaboratory Wiki' is the main interface to access all other Collaboratory service provided by the EBRAINS research infrastructure."

File: instances/service/KGSearchUI.jsonld [hunk 6]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/KGSearchUI",
```
Comment (UlrikeS91) [inline instances/service/KGSearchUI.jsonld]: Should reflect the name of the instance: "/EBRAINSKnowledgeGraphSearchUI"

File: instances/service/ModelCatalog.jsonld [hunk 7]
```diff
@@ -0,0 +1,14 @@
+{
+  "@context": {
+    "@vocab": "https://openminds.ebrains.eu/vocab/"
+  },
+  "@id": "https://openminds.ebrains.eu/instances/service/ModelCatalog",
```
Comment (UlrikeS91) [inline instances/service/ModelCatalog.jsonld]: Should reflect the name of the instance: "/EBRAINSModelCatalog"

File: instances/service/KGSearchUI.jsonld [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+  "@type": "https://openminds.ebrains.eu/controlledTerms/Service",
+  "definition": "Web application for searching the EBRAINS Knowledge Graph",
```
Comment (UlrikeS91) [inline instances/service/KGSearchUI.jsonld]: "The 'EBRAINS Knowledge Graph Search User Interface', short KGS UI, is a web application for searching the EBRAINS Knowledge Graph."

File: instances/service/KGSearchUI.jsonld [hunk 9]
```diff
@@ -0,0 +1,14 @@
...
+  "description": "The Knowledge Graph Search User Interface makes data, models and software discoverable and easy to use. The user interface allows for free text searches and provides filters to narrow searches based on metadata that classifies data according to experimental method, data modality and species among other options. The query results are displayed as Dataset cards containing key information about the dataset, e.g. its metadata, terms of use, and how to cite and reuse the data. Datasets that have undergone spatial registration to an atlas contain links to 3D or 2D viewers for navigating the data in atlas space.",
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "EBRAINS Knowledge Graph Search UI",
+  "preferredOntologyIdentifier": null,
+  "synonym": null
```
Comment (UlrikeS91) [inline instances/service/KGSearchUI.jsonld]: EBRAINS Knowledge Graph Search User Interface
EBRAINS KGS UI
Knowledge Graph Search User Interface
KGS UI

**Review by UlrikeS91 — CHANGES_REQUESTED** [review]

File: instances/service/AllenInstituteCellTypesDataPortal.jsonld (hunk 1 above)
Comment (apdavison) [reply instances/service/AllenInstituteCellTypesDataPortal.jsonld]: fixed

File: instances/service/CollaboratoryLab.jsonld (hunk 2 above)
Comment (apdavison) [reply instances/service/CollaboratoryLab.jsonld]: changed accordingly

File: instances/service/CollaboratoryLab.jsonld (hunk 3 above)
Comment (apdavison) [reply instances/service/CollaboratoryLab.jsonld]: fixed

File: instances/service/CollaboratoryWiki.jsonld (hunk 4 above)
Comment (apdavison) [reply instances/service/CollaboratoryWiki.jsonld]: fixed

File: instances/service/KGSearchUI.jsonld (hunk 6 above)
Comment (apdavison) [reply instances/service/KGSearchUI.jsonld]: fixed

File: instances/service/KGSearchUI.jsonld (hunk 8 above)
Comment (apdavison) [reply instances/service/KGSearchUI.jsonld]: changed accordingly

File: instances/service/KGSearchUI.jsonld (hunk 9 above)
Comment (apdavison) [reply instances/service/KGSearchUI.jsonld]: implemented

File: instances/service/ModelCatalog.jsonld (hunk 7 above)
Comment (apdavison) [reply instances/service/ModelCatalog.jsonld]: fixed

File: instances/service/CollaboratoryWiki.jsonld (hunk 5 above)
Comment (apdavison) [reply instances/service/CollaboratoryWiki.jsonld]: changed accordingly

**Comment by apdavison** [comment]
Thank you for the review, @UlrikeS91 !

**Review by UlrikeS91 — APPROVED** [review]
I would be fine with merging now. @lzehl would you like to look through them yourself as well?

File: instances/service/KGSearchUI.jsonld [hunk 10]
```diff
@@ -0,0 +1,19 @@
...
+  "description": "The Knowledge Graph Search User Interface makes data, models and software discoverable and easy to use. The user interface allows for free text searches and provides filters to narrow searches based on metadata that classifies data according to experimental method, data modality and species among other options. The query results are displayed as Dataset cards containing key information about the dataset, e.g. its metadata, terms of use, and how to cite and reuse the data. Datasets that have undergone spatial registration to an atlas contain links to 3D or 2D viewers for navigating the data in atlas space.",
+  "interlexIdentifier": null,
+  "knowledgeSpaceLink": null,
+  "name": "EBRAINS Knowledge Graph Search UI",
+  "preferredOntologyIdentifier": null,
+  "synonym": [
+    "EBRAINS Knowledge Graph Search User Interface",
+    "EBRAINS KGS UI",
+    "Knowledge Graph Search User Interface",
+    "KGS UI"
```
Comment (lzehl) [inline instances/service/KGSearchUI.jsonld:19]: And EBRAINS KG Search UI

**Review by lzehl — APPROVED** [review]
@apdavison there is one synonym missing for the KG Search UI :sweat_smile: but otherwise this is good to go. Could we add though linkages to the homepages to all of web services in the descriptions (or definitions)?

**Review by lzehl — APPROVED** [review]

**Pull request merged**